From 131ab3a9d51ca14914a693e18f7f2961efba911e Mon Sep 17 00:00:00 2001
From: Kate Hsuan <hpa@redhat.com>
Date: Mon, 9 Sep 2024 14:03:57 +0800
Subject: [PATCH 9/9] up-polkit: Replace with G_DEFINE_TYPE_WITH_PRIVATE

Since no interface was inplemented in UpPolkit class so G_DEFINE_TYPE_WITH_PRIVATE
macro is batter.
---
 src/up-polkit.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/up-polkit.c b/src/up-polkit.c
index fadfc30..1ead7e7 100644
--- a/src/up-polkit.c
+++ b/src/up-polkit.c
@@ -43,8 +43,7 @@ struct UpPolkitPrivate
 #endif
 };
 
-G_DEFINE_TYPE_WITH_CODE (UpPolkit, up_polkit, G_TYPE_OBJECT,
-                         G_ADD_PRIVATE (UpPolkit))
+G_DEFINE_TYPE_WITH_PRIVATE (UpPolkit, up_polkit, G_TYPE_OBJECT)
 
 #ifdef HAVE_POLKIT
 /**
-- 
2.46.0

